ibis/backends/flink/utils.py

```diff
@@ -282,6 +282,17 @@ def translate_literal(op: ops.Literal) -> str:
             raise ValueError("NaN is not supported in Flink SQL")
         elif math.isinf(value):
             raise ValueError("Infinity is not supported in Flink SQL")
+        elif dtype.is_decimal():
+            # When PyFlink infers schema from `decimal.Decimal` objects,
+            # it will be `DecimalType(38, 18)`.
```

Comment:
It defaults to (10, 0), but using this, since at the `ibis.literal` seems to be constructed from `decimal.Decimal`.